neuralcoref/train/evaluator.py

```diff
@@ -134,7 +133,8 @@ def display_clusters(self, doc_idx=None):
     ########################
     def get_max_score(self, batch, debug=False):
         inputs, mask = batch
-        inputs = tuple(Variable(i, volatile=True) for i in inputs)
+        with torch.no_grad():
+            inputs = tuple(i for i in inputs)
```

Comment:
Two stuff here:
- Can remove this line `inputs = tuple(i for i in inputs)`
- the ` with torch.no_grad():` context will go around the forward pass and it should be like this:
```python
with torch.no_grad():
    scores = self.model(inputs, concat_axis=1)
    scores.masked_fill_(mask, -float('Inf'))
    _, max_idx = scores.max(dim=1) # We may want to weight the single score with coref.greedyness
```